tinodesdk/src/main/java/co/tinode/tinodesdk/Topic.java

```diff
@@ -1002,9 +1002,6 @@ protected PromisedReply<ServerMessage> publish(final Drafty content, Map<String,
             }
             head.put("mime", Drafty.MIME_TYPE);
             attachments = content.getEntReferences();
-        } else if (head != null) {
-            // Plain text content should not have "mime" header. Clear it.
-            head.remove("mime");
```

Comment:
Why remove this line?

Reply:
Yes, please check for mime being video. Otherwise the app will send plain-text messages with an invalid mime `text/x-drafty`.

Reply:
Done.